src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/ArpeggioExpandableBlank.qml

```diff
@@ -36,6 +36,7 @@ ExpandableBlank {
 
     title: model ? model.title : ""
 
+    height: implicitHeight
```

Comment:
You don't have to specify this for every component. Just remove this line `height: childrenRect.height` from the loader (PlaybackGeneralSettings.qml)